Sources/Unbox.swift

```diff
@@ -454,6 +454,21 @@ extension Array: UnboxableCollection {
     }
 }
 
+/// Extension making Set an unboxable collection
+extension Set: UnboxableCollection {
+    public typealias UnboxValue = Element
+  
+    public static func unbox<T: UnboxCollectionElementTransformer>(value: Any, allowInvalidElements: Bool, transformer: T) throws -> Set? where T.UnboxedElement == UnboxValue {
+        guard let array = value as? [T.UnboxRawElement] else {
```

Comment:
How about using the implementation for `Array` here, instead of having to duplicate this code. So the body of this method would be:

```swift
guard let array = try [UnboxValue].unbox(value: value, allowInvalidElements: allowInvalidElements, transformer: transformer) else {
    return nil
}

return Set(array)
```